package.json

```diff
@@ -3,17 +3,10 @@
   "version": "0.0.1",
   "description": "Smart Oracle that pushes prices online every hour or when the price difference with the last price is greater or equal than 2%",
   "scripts": {
-    "build": "yarn install && yarn compile && npx tsc",
-    "clean": "yarn hardhat clean && rm -rf node_modules && rm -rf dist",
```

Comment:
Should `rm -rf node_modules && rm -rf dist` still be present to be able to properly clean TS files?

Reply:
We could keep `rm -rf node_modules` but there is no `dist`. A folder `.tmp` and `yarn.lock` are also generated as part of the process, should we remove them as part of `clean` too ?

Reply:
I think we should then remove `node_modules` and `.tmp` folders. Not sure if yarn.lock needs to be removed. Isn't it usually also commited and part of the repository?

Reply:
I agree, updated to also remove the `.tmp` folder but not `yarn.lock`. The `yarn.lock` gives the guarantee that when application is deployed in production, the packages installed will be exactly the same as in local development.